_ml-commons-plugin/cluster-settings.md

```diff
@@ -186,3 +186,63 @@ plugins.ml_commons.native_memory_threshold: 90
 
 - Default value: 90
 - Value range: [0, 100]
+
+## Allow custom deployment plans
+
+When enabled, grants users the ability to deploy models to specific ML nodes according to that user's permissions.
+
+### Setting
+
+```
+plugins.ml_commons.allow_custom_deployment_plan: false
+```
+
+### Values
+
+- Default value: false
+- Value range: [false, true]
+
+## Enable auto redeploy
+
+Automatically redeploys deployed or partially deployed models upon cluster failure. If all ML nodes inside a cluster crash, the model switches to the `DEPLOYED_FAILED` state, and the model must be deployed manually.
+
+### Setting
+
+```
+plugins.ml_commons.model_auto_redeploy.enable: false
+```
+
+### Values
+
+- Default value: false
+- Value range: [false, true]
+
+## Set retires for auto redeploy
+
+Sets the limit for the number of times a deployed or partially deployed model will try and redeploy upon cluster failure.
```

Comment:
Model auto redeploy happens when new nodes joined the cluster, either it's a crashed node rejoin or a new node first join the cluster, is it better to state `..... model will try and redeploy upon nodes join/rejoin the cluster`?